src/ert/config/_read_summary.py

```diff
@@ -316,7 +316,8 @@ def _read_spec(
                         hour=hour,
                         minute=minute,
                         second=microsecond // 10**6,
-                        microsecond=microsecond % 10**6,
+                        # Due to https://github.com/equinor/ert/issues/6952
```

Comment:
Could you make this comment stand-alone? Otherwise I foresee a future where the below line will simply be deleted, as I assume that it is related to this?

Reply:
Fixed